src/components/utils/constants.js

```diff
@@ -0,0 +1,27 @@
+const descriptionValue = `Excepteur pariatur nulla in ulla
+mco aliqua ea cupidatat incididunt proident culpa sint non exercitation deserunt dolor. 
+Voluptate voluptate dolor reprehenderit laboris fugiat laboris consequat id ex ullamco ad duis elit. 
+Irure do sint sint do veniam aliqua consequat est deserunt ullamco. 
+Minim ut commodo exercitation deserunt sunt qui proident culpa ad ex tempor aliqua do irure. 
+Et ullamco deserunt minim adipisicing et est dolor.
+`;
+
+const selectOptions = Object.freeze([
```

Comment:
object.freeze must be an object, instead of an array
{
  MINS_15: {name: '15 minutes', value: 15},
  ...
}

Reply:
Done